### PR TITLE
color handling: control chord_color

### PR DESCRIPTION
**Pull request by gph82**
Here's another suggestion for controlling chord color. I have found it useful in my plots (see example below)

As before it's a suggestion ;)  All these inputs would work:
```
mpl_chord_diagram.chord_diagram(C, 
                                chord_colors="red",
                                #chord_colors=[1, 0, 0],
                                #chord_colors=["red","green","blue"],
                                #chold_colors=[[1, 0, 0],
                                              [0, 1, 0],
                                              [0, 0, 1]],
                               );
```
Here's an example that I find visually appealing:

image

Next up: text label placing (probably an va="middle" has to be set somewhere

### CODE REVIEW

**Comment by gph82** [comment]
Hey BTW, I see now that you can create Issues in your repo (before I could not). 

Since PRs are a bit "intrusive" IMHO, I can just code away on my fork and occasionally open an issue suggesting changes, if you prefer it that way

**Comment by Silmathoron** [comment]
Hi there, no worries, I don't personally see PRs as "intruding" so I'm perfectly happy with you making some.
What I find extremely annoying, though, is GitHub not notifying me when you do, because I would find it normal to be notified... anyways... will try to have a look at that soon, it does seem like a good idea!

~~PS: please mention me (@Silmathoron) next time you make a PR, in case I have not find where this notification issue comes from by next time...~~ (solved that, apparently forked repos do not default to notifying the owner of "all actvities")

**Comment by Silmathoron** [comment]
OK, I have a few proposals to clarify some of the new features:

* first of all, I worry about the interaction between the new ``chord_colors`` and ``use_gradient`` so I would propose to check that ``use_gradient`` is ``False`` is ``chord_colors`` is not ``None``
* second, there is no simple way to compute the number of chords, so I would restrict the possibility of the list of colors:
  - either to require it to be equal to the number of nodes and each node would set the color of its outgoing chords (as in the non-gradient examples [on the Readme](https://github.com/Silmathoron/mpl_chord_diagram)
  - or remove it altogether

Let me know what you prefer and I'll make a proposal directly on you branch for some changes.

**Comment by gph82** [comment]
- OK on checking `use_gradient`  and either raising Exception or simply ignoring `chord_colors`, your call.
- on the list length, I think that's what it does, here
https://github.com/Silmathoron/mpl_chord_diagram/pull/18/commits/69246527e44e041e0cc385258cbf978e4d09f1ff#diff-4d8b1effdacc7bdf9e8d13ce0445d2ed8efc9eda358286665f6ab672de16402dR154
 right?

**Comment by Silmathoron** [comment]
>    on the list length, I think that's what it does, [here](https://github.com/Silmathoron/mpl_chord_diagram/commit/69246527e44e041e0cc385258cbf978e4d09f1ff#diff-4d8b1effdacc7bdf9e8d13ce0445d2ed8efc9eda358286665f6ab672de16402dR154) right?

Ah, yes indeed. I only read the docstring and that was not what I understood from it. I'll change the docstring then.

**Comment by gph82** [comment]
yes, please do change it. My docstrings can end up being a bit convoluted ;)

**Comment by Silmathoron** [comment]
Proposed changes [here](https://github.com/gph82/mpl_chord_diagram/pull/1)

**Comment by Silmathoron** [comment]
Thanks for merging my changes! Could you pull master in and fix the conflicts so the PR can be merged? (hopefully it's just a docstring issue)

**Comment by gph82** [comment]
Hi, "Only those with write access to this repository can merge pull requests."

**Comment by Silmathoron** [comment]
yes, of course, I'll take care of merging this PR ;)

**Comment by gph82** [comment]
sure, I lost track of what fork I was on for a sec :)

File: chord_diagram.py
Comment (Silmathoron) [inline chord_diagram.py]:
```suggestion
                  use_gradient=False, chord_colors=None, show=False, **kwargs):
```

File: chord_diagram.py
Comment (Silmathoron) [inline chord_diagram.py]:
```suggestion
    chord_colors : str, RGB tuple, list, optional (default: None)
        Specify color(s) to fill the chords differently from the arcs.
        When the keyword is not used, chord colors default to the colomap given
        by `colors`.
        Possible values for `chord_colors` are:
         * a single color or RGB tuple, e.g. "red" or ``(1, 0, 0)``; all chords
           will have this color
         * a list of colors, e.g. ``["red","green","blue"]``, one per node.
           Each chord will get its color from its associated source node, or
           from both nodes if `use_gradient` is True.
    show : bool, optional (default: False)
        Whether the plot should be displayed immediately via an automatic call
        to `plt.show()`.
```

**Review by Silmathoron — COMMENTED** [review]
Just changing the argument order

**Comment by Silmathoron** [comment]
OK, will check the code on my computer ASAP to make sure that everything is fine, will try to do that today

**Review by Silmathoron — APPROVED** [review]
OK, LGTM! :+1:

**Pull request merged**
